Zoom and pan: More options in the settings and improved error output

daswer123 opened the pull request:
## Description

Thanks to @w-e-w  for the suggested improvements in the code.

1) Improved the error output, now they will accurately identify the problem.

2) Made Zoom and Adjust brush size optional, as there are certain details.

3) You can select one of the 3 buttons "Alt", "Shift", "Ctrl" for Zoom and Adjust brush size

4) Added small tips in the description of the hotkeys, which will help get rid of possible problems

5) Removed canvas_swap_controls, it is no longer needed

## Screenshots/videos:

New Options:
![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/22278673/9b36f7b7-e8a1-4eab-9a68-16750b1145f7)

Full settings menu
![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/22278673/934efa0e-fcc6-46ca-9660-6524460dac4d)

Accurate error messages

![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/22278673/72430056-264c-48c9-ae1f-408ce9c39a8e)

## Checklist:

- [x] I have read [contributing wiki page](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing)
- [x] I have performed a self-review of my own code
- [x] My code follows the [style guidelines](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing#code-style)
- [x] My code passes [tests](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Tests)

daswer123 committed:
Improved error output, improved settings menu

daswer123 committed:
remove console.log

lastYoueven commented on extensions-builtin/canvas-zoom-and-pan/javascript/zoom.js:
```diff
@@ -42,37 +42,124 @@ onUiLoaded(async() => {
+                if (
+                    (!isSpecialKey && !usedSingleKeys.has(hotkey)) ||
+                    (isSpecialKey && !usedSpecialKeys.has(hotkey))
+                ) {
```
It can be simpler of the 'if()'

daswer123 replied:
Yes, I'll fix the code and make it more compact in the coming days, for now it just works.

daswer123 replied:
Corrected, removed this piece of code and made it smaller and clearer

daswer123 committed:
Reworked the disabling of functions, refactored part of the code

daswer123 committed:
Formatting code with Prettier

daswer123 commented:
UPD.

Slightly changed the approach, now you can push off any extension function of your choice. Removed unnecessary code and changed tooltip class to avoid conflicts in future.

![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/22278673/4ca9d8dd-f97c-404e-9ddc-bb7acc7c606e)

![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/22278673/15556ce4-87a3-44c9-b895-1e0e938484a3)
